package.json

```diff
@@ -39,35 +39,36 @@
     "semver": "7.5.2"
   },
   "dependencies": {
-    "@algorandfoundation/algokit-utils": "^2.3.0",
-    "chalk": "^5.3.0",
+    "@algorandfoundation/algokit-utils": "^3.0.0-beta.3",
```

Comment:
Should we use a non beta, or will we wait to merge this when it’s out of beta?

Reply:
Yes this is just a test, we will coordinate a proper release with Neil on Monday 👍